docs/getting-started/quickstart/README.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Quickstart with Github Codespaces
+---
+
+# EventStoreDB Quickstart using GitHub Codespaces
+
+This quickstart will guide you through getting started with EventStoreDB using [GitHub Codespaces](https://github.com/features/codespaces).
+
+::: info
+GitHub Codespaces provides an instant and preconfigured development environment in your browser for this quickstart.
+:::
+
+## Objectives
+
+In this quickstart, you will:
+
+- Set up a Codespace development environment.
+- Start an EventStoreDB server using Docker.
+- Append an event to EventStoreDB with sample code.
+- View the appended event using the Admin UI.
+- Read the appended event with sample code using the EventStoreDB client.
+
+## Prerequisites
+
+Before starting, ensure you have the following:
+
+- A GitHub account to use GitHub Codespaces.
+- Basic knowledge of one of the development languages/platforms below.
+- Familiarity with command-line operations.
+
+## Step 1: Set up Your Codespace
+
+1. Choose one of development languages/plaforms below and click the Codespaces link:
+
+::: tabs#dev-language-platform
+
+@tab Select >
+
+
+@tab Python
+|Language/Platform|Link to Codespaces|
+|-----|--------|
+|<div style="display: flex; justify-content: center; align-items: center;"><img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/python/python-original-wordmark.svg" alt="Python" style="height: 64px;" /></div>|[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/EventStore/EventStoreDB-From-Scratch-Python?ref=stephen-getting-started&devcontainer_path=.devcontainer%2Fquickstart%2Fdevcontainer.json)|
+
+@tab Java
+|Language/Platform|Link to Codespaces|
+|-----|--------|
+|<div style="display: flex; justify-content: center; align-items: center;"><img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/java/java-original-wordmark.svg" alt="Python" style="height: 64px;" /></div>|[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/EventStore/EventStoreDB-From-Scratch-Java?ref=stephen-getting-started&devcontainer_path=.devcontainer%2Fquickstart%2Fdevcontainer.json)|
+
+@tab .NET
+|Language/Platform|Link to Codespaces|
+|-----|--------|
+|<div style="display: flex; justify-content: center; align-items: center;"><img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/dot-net/dot-net-plain-wordmark.svg" alt="Python" style="height: 64px;" /></div>|[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/EventStore/EventStoreDB-From-Scratch-.Net?ref=stephen-getting-started&devcontainer_path=.devcontainer%2Fquickstart%2Fdevcontainer.json)|
```

Comment:
The name of the repository does not follow conventions in the dotnet world 
`EventStoreDB-From-Scratch-.Net` should be `EventStoreDB-From-Scratch-dotnet` 
I've already told that but got no reaction back then

Reply:
Yeh, I've created a task for this. Will fix this some time.

https://eventstore-engineering.atlassian.net/jira/software/projects/ES/list?selectedIssue=ES-76

Reply:
Please @stktung : not "some time" , it will never happen, either we do it as part of this PR or not at all .

Reply:
"Never" is a rather strong word 😉 Planning to work on it this week but just requires some coordination. Will let you know when it's done.

Reply:
@ylorph the repo names have been changed